index.js

```diff
@@ -1,7 +1,20 @@
 const readline = require('readline-sync');
+const calc = require('./calculator');
 
 console.log('Welcome to the calculator!');
 
-console.log('Please enter some input:');
-const response = readline.prompt();
-console.log('You entered: ' + response);
+console.log('Enter the operator:');
+const operator = readline.prompt();
+
+console.log(`How many numbers do you want to ${operator} ?`);
+const c = parseInt(readline.prompt());
+
+const calculator = calc.create_calculator(operator);
+
+for (var i = 0; i < c; i++) {
+  console.log('Enter a number:');
+  const n = parseInt(readline.prompt());
+  calculator.add_number(n);
+}
```

Comment:
We should try to avoid single character names for variables, changing these to more readable names will make the code cleaner and easier to understand. Some suggestions for renames:
```
c -> total_numbers
i -> current_number
n -> entered_number
```